core/src/main/java/tech/tablesaw/io/csv/CsvReadOptions.java

```diff
@@ -139,6 +151,8 @@ public int maxCharsPerColumn() {
     private Integer maxNumberOfColumns = 10_000;
     private Character commentPrefix;
     private boolean lineSeparatorDetectionEnabled = true;
+    private boolean ignoreTrailingWhitespacesInQuotes = true;
```

Comment:
I would default these to `false` or `null`. That's both more inline with what I'd expect, I think it'd make it easier for users to understand that we just use the univocity defaults, and it'd keep backwards compatibility as discussed on the thread earlier